hugo/layouts/docs/section.html

```diff
@@ -0,0 +1,27 @@
+{{ define "title"}}{{ .Title }}{{ end }}
+{{ define "description" }} description: Keep up to date with our progress building Fluidkeys, a way
```

Comment:
Don't think this is supposed to be here!

Reply:
Whoops! 👍  I'll drop it and push!